stories/modules-tagmanager-settings.stories.js

```diff
@@ -149,70 +149,6 @@ storiesOf( 'Tag Manager Module/Settings', module )
 			decorators: [ withRegistry ],
 		}
 	)
-	.add(
```

Comment:
Rather than delete both these _View..._ stories, I think we should keep one of them - say the _with access_ one, remove the call to `receiveGetTagPermission` and remove `(with access)` from the description. Otherwise we don't have a story to show the "existing tag" behaviour at all.